src/modules/address-book/ModuleAddressBook.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <mew-address-select
       ref="addressSelect"
-      :resolved-addr="resolvedAddr"
+      :hint="resolvedAddr"
```

Comment:
don't you still need resolved-addr here? hint should just be for kleros and ens reverse name. resolve-addr is for ens names